launchpad/launchpad_deposit.gno

```diff
@@ -57,8 +60,17 @@ func DepositGns(
 		panic(ufmt.Sprintf("tier is not active: %s", tierStr))
 	}
 
+	common.IsHalted()
+	en.MintAndDistributeGns()
+
 	// after all pre-checks
 	calculateDepositReward()
+	project = projects[projectId]    // get updates project
+	tier = getTier(project, tierStr) // get updates tier
+
+	// ORIG NOT ORIG
```

Comment:
what does this mean?

Reply:
left over, cleaned up d52e746a8096c94641a42388c350c2b59bf4775a